### PR TITLE
Fractional NFT ownership DAO contract

### PR DESCRIPTION
**Pull request by emishur**

### CODE REVIEW

File: fractional/ligo/src/fractional_dao.mligo [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  admin  = {
+    admin = ("tz1YPSCGWXwBdTncK2aCctSZAXWvGsGwVJqU" : address);
+    pending_admin = (None : address option);
+    paused = false;
+  };
+  vote_nonce = 0n;
+  owned_nfts = (Big_map.empty : ownership);
+  pending_votes = (Big_map.empty : pending_votes);
+  metadata  = Big_map.literal [
+    ("", Bytes.pack "tezos-storage:content" );
```
Comment (smondet) [inline fractional/ligo/src/fractional_dao.mligo:291]: I guess this is actually never used, but I don't think it's good to confuse people by using `pack`.

File: fractional/ligo/src/fractional_dao.mligo [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+
+let mint_ownership_token (ownership, token_id, s : 
+    ownership_stake list * token_id * multi_token_storage) : multi_token_storage =
+  let meta = { 
+    token_id = token_id;
+    extras = Map.literal [
+    ("symbol", Bytes.pack "OT");
+    ("name", Bytes.pack "Ownership Token");
+    ("decimals", Bytes.pack "0");
+  ]; } in
```
Comment (smondet) [inline fractional/ligo/src/fractional_dao.mligo:99]: Indentation is kinda confusing there :)

**Review by smondet — APPROVED** [review]
LGTM, just minor things

File: fractional/ligo/src/fractional_dao.mligo (hunk 1 above)
Comment (emishur) [reply fractional/ligo/src/fractional_dao.mligo:291]: Oh, I actually use it to run ligo off-chain `compile-storage` command to generate Michelson for contract origination.

File: fractional/ligo/src/fractional_dao.mligo (hunk 2 above)
Comment (emishur) [reply fractional/ligo/src/fractional_dao.mligo]: fixed

**Pull request merged**